core/src/main.rs

```diff
@@ -23,8 +23,6 @@ use tauri::Manager;
 
 lazy_static! {
     pub static ref ARK_SHELF_WORKING_DIR: PathBuf = PathBuf::from(Cli::parse().path);
-    pub static ref ARK_SHELF_DATA_PATH: PathBuf =
-        PathBuf::from(Cli::parse().path).join(".ark").join("shelf");
```

Comment:
This last string, `"shelf"` can be changed.

So far, the best source to see the list of magic strings is here:
https://github.com/ARK-Builders/arklib-android/blob/main/lib/src/main/java/space/taran/arklib/ArkFiles.kt#L14

We will move it into Rust `arklib`, it's just the current state that we focused more on mobile before.